### ISSUE TITLE
Feature Request - Improve transferring items between submarines

### ISSUE THREAD

**Issue by Zetabite** [Regalis11/Barotrauma#8657]
Currently, you need to pack up stuff in containers that you can carry. You are limited by the size of that containers and size of the crew. Carrying items inside a station causes them to despawn on submarine switch.
Now, to how that could actually be solved. Since I havent played a whole lot of Baro, I cant tell if there is a use for the empty room inside a station, adjacent to the room where one switches the submarine. If that room is without use yet, there could be a terminal, that allows inserting crates/items for transfer between subs. On a subs departure, those crates/items will be transferred into the current vessel.

Additional Notes:
Also, there should be a warning if items are dropped inside a station, since they will despawn on submarine switch. This costed me a run already, even tho a minor addition in its gameplay nature, it would help a lot of people preventing loosing their save game, especially since the game only provides one save slot per run.
Another issue is, that crates are rather awkwardly available, so getting large enough storage to transfer items between ships is an additional hurdle. Since you get boxes for free when buying stuff, they should at least be craftable. This makes them not highly available, and prevents from people stocking up on them in large amounts. In this manner they stay free, while not becoming a cashcow.

**Comment by LePiromano** [comment]
there's a kinda similar suggestion in discussions about this issue
https://github.com/Regalis11/Barotrauma/discussions/6601

**Comment by Zetabite** [comment]
yup, i agree, didnt see that one. Hopefully this suggestion adds to the idea.
